example/src/Pages/Layouts.js

```diff
@@ -25,104 +52,191 @@ const Layouts = () => {
   }, []);
 
   return (
-    <Container>
-      <Header
-        title="Layouts"
-        actionBlock={<Button label="Primary Action" />}
-        breadcrumbs={[{ text: "Home", link: "/" }]}
-      />
-      <SubHeader
-        searchProps={{
-          value: searchString,
-          onChange: (e) => setSearchString(e.target.value),
-        }}
-        deleteButtonProps={{
-          count: 0,
-          selectedIDs: [],
-          onClick: () => {},
-        }}
-        disableButtonProps={{
-          count: 0,
-          selectedIDs: [],
-          onClick: () => {},
-        }}
-      />
-      <Scrollable className="w-full">
-        {isLoading ? (
-          <PageLoader />
-        ) : (
-          <table
-            className={`v2-nui-table v2-nui-table--checkbox v2-nui-table--actions`}
+    <div className="flex">
+      <MenuBar
+        showMenu={showMenu}
+        title={
+          <div className="flex justify-between">
+            <Typography style="h2">Contacts</Typography>
+          </div>
+        }
+      >
+        <MenuBar.Block label="All" count={13} active />
+        <MenuBar.Block label="Users" count={2} />
+        <MenuBar.Block label="Leads" count={7} />
+        <MenuBar.Block label="Visitors" count={4} />
+
+        <MenuBar.SubTitle
+          iconProps={[
+            {
+              icon: () => <Search size={20} />,
+            },
+          ]}
+        >
+          <Typography
+            component="h4"
+            style="h5"
+            textTransform="uppercase"
+            weight="bold"
+          >
+            Segments
+          </Typography>
+        </MenuBar.SubTitle>
+        <MenuBar.Search />
```

Comment:
@ajmaln _a Can you make Search a controller component to test out the changes.

Reply:
You can also provide extra prop `collapse` in order to collapse the Search field.